analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTest.cs

```diff
@@ -100,47 +101,46 @@ public void SetParameterValues_WhenGivenValidSonarLintFileAndDoesNotContainAnaly
         }
 
         [TestMethod]
-        public void SetParameterValues_CalledTwiceAfterChangeInConfigFile_UpdatesProperties()
+        public void SetParameterValues_WithNonExistingPath_UsesInMemoryText()
```

Comment:
Some comparison alignment problems here due to a mistake on my part:
This test replaces the previous `SetParameterValues_WithBadPath_DoesNotPopulateProperties` which was deleted below.
`SetParameterValues_CalledTwiceAfterChangeInConfigFile_UpdatesProperties` is modified just after this one.

Reply:
`...NonExistentPath...`